### ISSUE TITLE
Joyride mounts to the DOM but the beacon is not visible

### ISSUE THREAD

**Issue by sdelpuerto-intelygenz** [gilbarbara/react-joyride#387]
>Expected behavior

When I render the Joyride component, a beacon shows up.

>Actual behavior

No beacon appears on the screen, even though it exists on the DOM as I checked on the browser inspector.

>Steps to reproduce the problem

```
// inside render

const steps = [{
  title: 'Title,
  content: 'Content,
  placement: 'top-start',
  target: 'body'
}]

return (
  <div>
    <Joyride steps={steps} run />
  </div>
)
```

>React version

15.3.2

>React-Joyride version

2.0.0-11

>Error stack

No errors, the beacon simply doesn't appear visible.

With react-joyride version 1 the beacon rendered correctly, so I'm guessing this has something to do with the styles varying slightly with version 2. If I manually change the CSS visibility property of the parent element of joyride-beacon to "visible" instead of "hidden", then the beacon appears, but it's buggy (it doesn't disappear when opening the tooltip).

![image](https://user-images.githubusercontent.com/37080276/41280403-21a0aa66-6e2f-11e8-9e02-2585d6175f30.png)

I would appreciate if you could take a look into this.

**Comment by keul** [comment]
Probably duplicate of #361 (you can also find some workaround there)

**Comment by gilbarbara** [comment]
Fixed in `react-joyride@2.0.0-12`

**Comment by afrancht** [comment]
I'm still having this issue and I'm on `2.2.1`

**Comment by gilbarbara** [comment]
hey @afrancht 

Why do you have two instances of joyride mounted?
Anyway, it's impossible to debug without seeing the code running. Probably something with z-index? Check your code and tweak the z-index.

**Comment by afrancht** [comment]
Excuse my terrible post. I should've attached a code snippet. I have figured it out:

1. As you said I should play with the z-index
2. I had to make sure that everything was rendered before using the selector, thus I had to add joyride to the "highest" order component.

I think it would be worthy including this in the quick setup guide so that people can get it running. It is intuitive to get it to run on a new project but on an old one, you might want to include that information when using it with react.

Sorry for the bad post and thank you for the quick reply!

**Comment by erkanonal** [comment]
> Fixed in `react-joyride@2.0.0-12`

Hey there!,
I am having the same issue on 2.2.1. Although I use the disableBeacon: true, beacon is not visible but still in the dom. Is this intentional? If it is disable, it should not be in the dom in the first place right? I am able to solve my problem via changing the css of beacon but I am just asking.
